model_garden/deploy/list_deployable_models_example.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Vertex AI sample for listing deployable models in
+    Model Garden.
+"""
+
+from typing import List
+
+
+def list_deployable_models(list_hugging_face_models_only : bool = False, model_filter : str = "") -> List[str]:
```

Comment:
```suggestion
def list_deployable_models() -> List[str]:
```
You can simple the `def` if you can use 


```
deployable_models = model_garden.list_deployable_models(list_hf_models=False, model_filter="gemma")
```